backend/user_data/settings/plots_default.yaml

```
@@ -0,0 +1,8 @@
+custom_font: ''
+file_format: png
```

```suggestion
file_format: svg
```
Das war die eine Änderung, die ich gerade noch ins alte PROTzilla gemerged habe. ^^